text/0000-safer-transmute.md

```diff
@@ -643,93 +532,73 @@ Even with `NeglectValidity`, the compiler will statically reject transmutations
 
 #[repr(C)] enum Bar { Z = 42 }
 
-let _ = <Bar as TransmuteFrom<Foo, Here!(), NeglectValidity>::unsafe_transmute_from(Foo::N) // Compile error!
+let _ = <Bar as TransmuteFrom<Foo, NeglectValidity>::unsafe_transmute_from(Foo::N) // Compile error!
 ```
 
-# Reference-level explanation
-[reference-level-explanation]: #reference-level-explanation
-
-## Implementation Guidance
-Three items in this RFC require special compiler support:
-  - `Here!()`
-  - `Constructible` (a private implementation detail of `TransmuteFrom`)
-  - `TransmuteFrom`
-
-### Implementing `Here!()`
-The `Here!` macro produces a type that uniquely identifies its invocation scope. For instance:
-```rust
-use static_assertions::*;
-
-type A = Here!();
-type B = Here!();
+#### `NeglectSafety`
+By default, `TransmuteFrom`'s methods require that all instantiations of the source type are `Muckable`. If the destination type is a mutate-able reference, the source type must *also* be `Muckable`. This precludes transmutations that are [well-defined][sound transmutation] but not [safe][safe-and-stable transmutation].
```

Comment:
What does "all instantiations of the source type" mean? Is this any different from "requires that the source type be `Muckable`"?

Reply:
This should read:
```suggestion
By default, `TransmuteFrom`'s methods require that the destination type is `Muckable`. If the destination type is a mutate-able reference, the source type must *also* be `Muckable`. This precludes transmutations that are [well-defined][sound transmutation] but not [safe][safe-and-stable transmutation].
```